### PR TITLE
Fix quantity formatter initialization

### PR DESCRIPTION
**Pull request by bsteinbk**
Ensure quantity formatter is initialized after IModelApp.startup is awaited.

### CODE REVIEW

**Comment by bsteinbk** [comment]
@mergifyio backport release/2.11.x

File: core/frontend/src/IModelApp.ts [hunk 1]
```diff
@@ -445,12 +445,14 @@ export class IModelApp {
     });
 
     // process async onInitialized methods
+    const initPromises: Promise<void>[] = [];
```
Comment (pmconne) [inline core/frontend/src/IModelApp.ts:448]: Why not make this a one-liner: `await this.quantityFormatter.onInitialized()`?

**Review by pmconne — CHANGES_REQUESTED** [review]

File: core/frontend/src/IModelApp.ts (hunk 1 above)
Comment (bsteinbk) [reply core/frontend/src/IModelApp.ts:448]: I was just trying to provide a location for this and any future async initializations.... but I can change to it to a single line if you prefer.

Comment (pmconne) [reply core/frontend/src/IModelApp.ts:448]: No reason to make it ugly and convoluted until we have more than one thing to initialize.

Comment (bsteinbk) [reply core/frontend/src/IModelApp.ts]: OK, I de-uglified the change.

**Review by pmconne — APPROVED** [review]

**Review by NancyMcCallB — APPROVED** [review]

**Pull request merged**